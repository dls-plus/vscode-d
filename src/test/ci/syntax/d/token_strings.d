auto x = q{hello
world { great! int x = 4;
}
};