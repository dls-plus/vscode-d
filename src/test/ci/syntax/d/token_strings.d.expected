[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":6,"scope":"variable.d"},{"start":6,"end":7,"scope":"source.d"},{"start":7,"end":8,"scope":"keyword.operator.assign.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":11,"scope":"string.quoted.token.d"},{"start":11,"end":16,"scope":"variable.d"}]
[{"start":0,"end":5,"scope":"variable.d"},{"start":5,"end":8,"scope":"source.d"},{"start":8,"end":13,"scope":"variable.d"},{"start":13,"end":14,"scope":"keyword.operator.logical.d"},{"start":14,"end":15,"scope":"source.d"},{"start":15,"end":18,"scope":"storage.type.basic-type.d"},{"start":18,"end":19,"scope":"source.d"},{"start":19,"end":20,"scope":"variable.d"},{"start":20,"end":21,"scope":"source.d"},{"start":21,"end":22,"scope":"keyword.operator.assign.d"},{"start":22,"end":23,"scope":"source.d"},{"start":23,"end":24,"scope":"constant.numeric.integer.decimal.d"},{"start":24,"end":26,"scope":"meta.statement.end.d"}]
[{"start":0,"end":2,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":3,"scope":"meta.statement.end.d"}]